hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -432,45 +433,55 @@
     {
       name: chip_rv_dm_cpu_debug_mem
       desc: '''Verify access to the debug mem from the CPU.
+
+            - CPU programs RW type CSRs (such as progbuf0 and progbuf1) with random value.
+            - Read back to check their values for correctness.
             '''
       milestone: V2
       tests: []
     }
     {
       name: chip_rv_dm_jtag_debug_mem
       desc: '''Verify access to the debug mem from the external JTAG interface.
+
+            - Load debug ROM and read back through jtag to check the values for correctness.
```

Comment:
I'm not quite sure what this test was meant to test for. The debug *ROM* is neither readable not writable through JTAG. The program buffer, which is part of the debug *RAM* is readable and writable through JTAG, but already tested below in the test that's currently named chip_rv_dm_cpu_debug_req.

Probably just remove this test?

https://github.com/pulp-platform/riscv-dbg/blob/master/doc/debug-system.md#debug-memory for more information on the debug memory and its organization.

Reply:
Thanks for correcting me. Removed this test.